File: go/1131.go
package main

import "fmt"

func main() {
  var grenais, golInt, golGre, inter, gremio, emp, choose int
  var winner string

  for {
    fmt.Scanf("%d %d", &golInt, &golGre)
    if golInt > golGre {
      inter += 1
    }
    
    if golGre > golInt {
      gremio += 1
    }

    if golGre == golInt {
      emp += 1
    }

    grenais += 1

    fmt.Println("Novo grenal (1-sim 2-nao)")
    fmt.Scanf("%d", &choose)

    if choose == 2 {
      break
    }
  }

  fmt.Printf("%d grenais\nInter:%d\nGremio:%d\nEmpates:%d\n", grenais, inter, gremio, emp)

  if inter == gremio {
    fmt.Println("Nao houve vencedor")
    return
  }

  winner = "Inter"
  if gremio > inter {
    winner = "Gremio"
  }

  fmt.Printf("%s venceu mais\n", winner)
}

File: lua/1131.lua
local grenais, golInt, golGre, inter, gremio, emp, choose = 0, 0, 0, 0, 0, 0, 0
local winner = ""

while true do
    local golInt = io.read("*n")
    local golGre = io.read("*n")

    if golInt > golGre then
        inter = inter + 1
    elseif golGre > golInt then
        gremio = gremio + 1
    else
        emp = emp + 1
    end

    grenais = grenais + 1

    print("Novo grenal (1-sim 2-nao)")
    local choose = io.read("*n")

    if choose == 2 then
        break
    end
end

print(string.format("%d grenais", grenais))
print(string.format("Inter:%d", inter))
print(string.format("Gremio:%d", gremio))
print(string.format("Empates:%d", emp))

if inter == gremio then
    print("Nao houve vencedor")
else
    winner = "Inter"
    if gremio > inter then
        winner = "Gremio"
    end

    print(string.format("%s venceu mais", winner))
end
